samples/java_springboot/03.welcome-user/src/main/java/com/microsoft/bot/sample/welcomeuser/WelcomeUserBot.java

```diff
@@ -37,27 +46,27 @@
  */
 public class WelcomeUserBot extends ActivityHandler {
     // Messages sent to the user.
-    private static final String WELCOMEMESSAGE =
+    private static final String WELCOME_MESSAGE =
         "This is a simple Welcome Bot sample. This bot will introduce you "
             + "to welcoming and greeting users. You can say 'intro' to see the "
             + "introduction card. If you are running this bot in the Bot Framework "
             + "Emulator, press the 'Start Over' button to simulate user joining "
             + "a bot or a channel";
 
-    private static final String INFOMESSAGE =
+    private static final String INFO_MESSAGE =
         "You are seeing this message because the bot received at least one "
             + "'ConversationUpdate' event, indicating you (and possibly others) "
             + "joined the conversation. If you are using the emulator, pressing "
             + "the 'Start Over' button to trigger this event again. The specifics "
             + "of the 'ConversationUpdate' event depends on the channel. You can "
             + "read more information at: " + "https://aka.ms/about-botframework-welcome-user";
 
-    private String LOCALEMESSAGE =
+    private String LOCALE_MESSAGE =
```

Comment:
```suggestion
    private String localeMessage =
```

As this is not `static`